docs/reference/release-notes/7.17.5.asciidoc

```diff
@@ -3,23 +3,107 @@
+Highlighting::
+* `FastVectorHighlighter` should use `ValueFetchers` to load source data {es-pull}85815[#85815] (issues: {es-issue}75011[#75011], {es-issue}84690[#84690], {es-issue}82458[#82458], {es-issue}80895[#80895])
+* `FastVectorHighlighter` should use `ValueFetchers` to load source data {es-pull}87445[#87445] (issues: {es-issue}75011[#75011], {es-issue}84690[#84690], {es-issue}82458[#82458], {es-issue}80895[#80895])
+
+ILM+SLM::
+* Fix `max_primary_shard_size` resize factor math {es-pull}86897[#86897]
+
+Indices APIs::
+* Make `GetIndexAction` cancellable {es-pull}87681[#87681]
+
+Infra/Circuit Breakers::
+* Make CBE message creation more robust {es-pull}87881[#87881]
+
+Infra/Core::
+* Adjust osprobe assertion for burst cpu {es-pull}86990[#86990]
+* Clean up `DeflateCompressor` after exception {es-pull}87163[#87163] (issue: {es-issue}87160[#87160])
+* Prevent migration of indices that match templates {es-pull}87933[#87933] (issues: {es-issue}86801[#86801], {es-issue}87827[#87827])
+
+Infra/Scripting::
+* Script: Load Whitelists as Resource {es-pull}87539[#87539]
+
+Ingest::
+* Don't download geoip databases if geoip system index is blocked {es-pull}86842[#86842]
+* Don't ignore pipeline for upserts in bulk api {es-pull}87719[#87719] (issue: {es-issue}87131[#87131])
+
+Java Low Level REST Client::
+* Do not retry client requests when failing with `ContentTooLargeException` {es-pull}87248[#87248] (issue: {es-issue}86041[#86041])
+
+Machine Learning::
+* Adjacency weighting fixes in categorization {ml-pull}2277[#2277]
+* Fix a source of "Discarding sample = nan, weights = ..." log errors for time series
+  anomaly detection {ml-pull}2286[#2286]
+* Fix ML task auditor exception early in cluster lifecycle {es-pull}87023[#87023] (issue: {es-issue}87002[#87002])
+
+Recovery::
+* Do not leak recover from snapshot file permits when recover from snapshot is disabled {es-pull}87633[#87633] (issue: {es-issue}86705[#86705])
+* Fail shard if STARTED after master failover {es-pull}87451[#87451] (issue: {es-issue}87367[#87367])
+
+Search::
+* Handle empty point values in `DiskUsage` API {es-pull}87826[#87826] (issue: {es-issue}87761[#87761])
+* Make sure to rewrite explain query on coordinator {es-pull}87013[#87013] (issue: {es-issue}64281[#64281])
+
+Security::
+* Security plugin close releasable realms {es-pull}87429[#87429] (issue: {es-issue}86286[#86286])
+
+Snapshot/Restore::
+* Fork after calling `getRepositoryData` from `StoreRecovery` {es-pull}87254[#87254] (issue: {es-issue}87237[#87237])
+* Fork after calling `getRepositoryData` from `StoreRecovery` {es-pull}87264[#87264] (issue: {es-issue}87237[#87237])
+* Throw exception on illegal `RepositoryData` updates {es-pull}87654[#87654]
+
+Stats::
+* Run `TransportClusterInfoActions` on MANAGEMENT pool {es-pull}87679[#87679]
+
+Transform::
+* Execute `_refresh` (with system permissions) separately from `delete by query` (with user permissions) {es-pull}88005[#88005] (issue: {es-issue}88001[#88001])
```

Comment:
I don't think this should be here.  7.17.5 was commit 8d61b4f7ddf931f219e3745f295ed2bbc50c8e84, this was eb8fee10aef35d018dc6e7060844a7cfca0991f5, and if you look at the commit history eb8fee10aef35d018dc6e7060844a7cfca0991f5 comes after 8d61b4f7ddf931f219e3745f295ed2bbc50c8e84. This will be in 7.17.6. I'll relabel the PR.